inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -571,22 +571,34 @@ def normalize_journal_titles(obj, eng):
     Returns:
        None
     """
-    publications = obj.data.get('publication_info')
-
-    if not publications:
-        return None
-
-    for index, publication in enumerate(publications):
-        if 'journal_title' in publication:
-            normalized_title = normalize_journal_title(publication['journal_title'])
-            obj.data['publication_info'][index]['journal_title'] = normalized_title
-            ref_query = RecordMetadata.query.filter(
-                RecordMetadata.json['_collections'].op('?')('Journals')).filter(
-                cast(RecordMetadata.json['short_title'], String) == type_coerce(normalized_title, JSON))
-            result = db.session.execute(ref_query).fetchone()
-
-            if result:
-                obj.data['publication_info'][index]['journal_record'] = result.records_metadata_json['self']
+    publications = obj.data.get('publication_info', [])
+
+    for publication in publications:
+        normalize_journal_title_entry(publication)
+
+    references = obj.data.get("references", [])
+    for reference in references:
+        publication_info = get_value(reference, 'reference.publication_info')
+        if not publication_info:
+            continue
+        normalize_journal_title_entry(publication_info)
+
+
+def normalize_journal_title_entry(publication_info):
+    if 'journal_title' not in publication_info:
+        return None, None
```

Comment:
why are you returning this? the main branch doesn't explicitly return anything (it mutates the input)
```suggestion
        return
```

Reply:
You are right. Leftover from time when this was returning modified title and record.